src/ringme.js

```diff
@@ -97,6 +99,29 @@ const RingMe = new function () {
     return img
   }
 
+  const _styleButton = function (button) {
+    const box =
+      'border-radius: 6px;' +
+      'padding: 3px 23px 3px 46px;'
+    const genericBackground =
+      'background-color: #3bc1d3;'
+    const dualBackground =
+      'background: ' +
+        'url(../assets/button.svg) 6px center / 20px 20px no-repeat,' +
+        'linear-gradient(to right, #3bc1d3, #3bc1d3) left top / 32px auto repeat-y,' +
+        'linear-gradient(to right, #75d3e0, #75d3e0) 23px top / 20px auto repeat;'
+    const text =
+      'color: #fff;' +
+      'font-family: Georgia, Cambria, "Times New Roman", Times, serif;' +
+      'font-weight: bold;' +
+      'text-decoration: none;'
+
+    const styledButton = button
```

Comment:
@samnegri it's kind of counter intuitive for me to have a `const` object instance and then _change it's state_?!

I would have naturally use `let` on this line, but since it _works_ and `const` seems to be the new kid on the block, I did use `const`.

What's your input on this?